component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/TOTPAuthenticator.java

```diff
@@ -838,4 +836,27 @@ private String buildAbsoluteURL(String redirectUrl) throws URISyntaxException, U
 		}
 		return ServiceURLBuilder.create().addPath(redirectUrl).build().getAbsolutePublicURL();
 	}
+
+	private long calculateUnlockTime(long unlockTimePropertyValue, double unlockTimeRatio,
+									 int failedLoginLockoutCountValue) {
+
+		if (isIndefiniteLock(unlockTimePropertyValue)) {
+			return 0;
+		} else {
```

Comment:
We can remove the else block and just add the return statement right, for simplicity and readability.

Reply:
Resolved with https://github.com/wso2-extensions/identity-outbound-auth-totp/pull/192/commits/6f7f99bf44e427563988e53e0932959dbe9315ba